docs/gro-ontology.md

```diff
@@ -0,0 +1,119 @@
+# Gro Ontology
+Gro has brought the world of agricultural data into a single ontology. This ontology classifies all data into a limited set of entities. 
+
+The entity types around which the data is organized in Gro are:
+
+* items -- what the data is about (e.g., Rice, Sugar, Rainfall, etc.).
+* metrics -- how the data is measured (e.g., Production, Precipitation, etc.).
+* regions -- where the data is from/about (e.g., a country, a weather station, a city etc.). Some data such as trade flows might have a partner region.
+* frequencies -- e.g., daily, weekly etc.
+* sources -- the organization or sub-organization that generates the data (e.g., USDA PS&D).
+* units -- the units in which the data is presented (e.g., tonnes, liters, etc.).
+
+## Entity fields
+Each specific entity will have an `id`, `name`, and various other properties. For example, "Soybeans" is a specific item with `id = 270` and the following information
+```py
+{'allowedAggregations': None,
+ 'contains': [1737,
+  7401,
+  7578,
+  7579,
+  5136,
+  5021,
+  5135,
+  5137,
+  12418,
+  5776,
+  5777,
+  3035,
+  3036,
+  9312,
+  9284,
+  6330,
+  7577,
+  12728,
+  12729,
+  12730,
+  12731,
+  12732,
+  12733,
+  12734,
+  12735,
+  12736,
+  12737,
+  10134],
+ 'definition': 'The seeds and harvested crops of plants belonging to the species <i>Glycine max</i> that are used in the production of oil and both human and livestock consumption.',
+ 'id': 270,
+ 'name': 'Soybeans',
+ 'rankingScore': 2.204}

```

Comment:
Let's remove `rankingScore`. Just an internal property and we're not explaining publicly what it is/how it works.

Reply:
done